cmd/frontend/graphqlbackend/search_results.go

```diff
@@ -1205,7 +1206,7 @@ func (r *searchResolver) resultsWithTimeoutSuggestion(ctx context.Context) (*Sea
 	// type searchers (file, diff, symbol, etc) completely timed out and could not
 	// produce even partial results. Other searcher types may have produced results.
 	//
-	// In this case, or if we got a partial timeout where ALL repositories timed out,
+	// In this case, or if we got a partial timeout where ALL searchrepos timed out,
```

Comment:
Overzealous auto-refactor. 
```suggestion
	// In this case, or if we got a partial timeout where ALL repositories timed out,
```